README.md

```diff
@@ -79,6 +79,7 @@ The following 34 devices are supported.
 | [MMA8653 accelerometer](https://www.nxp.com/docs/en/data-sheet/MMA8653FC.pdf) | I2C |
 | [MPU6050 accelerometer/gyroscope](https://store.invensense.com/datasheets/invensense/MPU-6050_DataSheet_V3%204.pdf) | I2C |
 | [PCD8544 display](http://eia.udg.edu/~forest/PCD8544_1.pdf) | SPI |
+| [Shift register](https://en.wikipedia.org/wiki/Shift_register) | GPIO |
```

Comment:
Since your current implementation is parallel-in serial out, perhaps https://en.wikipedia.org/wiki/Shift_register#Parallel-in_serial-out_(PISO) is a more correct link.

Also might be good to mention `74HC165` and `74165` somewhere which are the most common numbers for this chip, even though I think they are all clones now.